docs/modules/getting_started/pages/basic_walkthrough.adoc

```diff
@@ -186,3 +153,18 @@ For nodes, you can configure only a table graph.
 ** (Optional) Configure the settings in the bottom half of the panel to customize the graph.
 . Update the panel's *Name*.
 . Click *Save* at the top-right of the panel.
+
+== Use data links to provide alarm context
+
+Data links in Grafana let you create a user-defined operation (for example, a link to a node detail page or a knowledge base article, links based on alarm severity, and so on) that might help users diagnose an alarm.
+Once created, a link to the specified data will appear when you right-click an alarm in an alarms table:
+
+image::gf-data-link.png["Alarms table with three alarms. The context menu is displayed, listing three defined options."]
```

Comment:
This is an image of the Alarm Table OpenNMS context menu, but not the Grafana Data Link context menu.